### ISSUE TITLE
Using Ace on Django Framework.

### ISSUE THREAD

**Issue by georgerobert** [ajaxorg/ace#4749]
Hello! I am currently trying to implement the Ace editor onto my website that I am developing using the Django framework. My main issue is that I cannot make the scripts in the html file to recognize the files from the ace repository. I followed the steps from the website and other online tutorials but I cannot make it work. The code I have written is:
```
<html lang="en">
<head>
<title>ACE in Action</title>
<style type="text/css" media="screen">
    #editor {
        position: absolute;
        top: 0;
        right: 0;
        bottom: 0;
        left: 0;
    }
</style>
</head>
<body>

<div id="editor">function foo(items) {
    var x = "All this is syntax highlighted";
    return x;
}</div>

<script src="src/ace.js"></script>
<script>
    var editor = ace.edit("editor");
    editor.setTheme("ace/theme/monokai");
    editor.session.setMode("ace/mode/javascript");
</script>
</body>
</html>
```
which is the exact code from the website where I modified the src directory in the script for the ace.js file to match my project. The structure of my project is as follows:

Main App Folder / Template Folder / Ace src folder from the repository 
Main App Folder / Template Folder / html file for the actual page

Basically I have both the src folder for ace and the html file in a template folder.

The error I get is a 404 Error in the terminal that says:
 
**Not Found: /src/ace.js               
[29/Apr/2021 15:49:19] "GET /ace/ace.js HTTP/1.1" 404 2113**  

I am fairly new to html and the Django framework so I do not know if this is my mistake or something related to Django. If someone could help me out I would be very grateful!
